Nasal/electrical.nas

```diff
@@ -79,7 +84,7 @@ BatteryClass.new = func {
 #
 
 BatteryClass.apply_load = func (amps, dt) {
-    var old_charge_percent = getprop("/systems/electrical/battery-charge-percent");
+    var old_charge_percent = me.charge_percent;
```

Comment:
I did these two modifications to `BatteryClass.apply_load` after comparing it with the same function in the c150. For some reason, our battery was constantly stuck at 24.5 V even though the percentage was draining, and even when the percentage was at 0% the systems would all work as if the battery was fully charged. I then tried changing this lines in the hope that they might fix the problem, and indeed they did.